ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/target/IUBundleContainer.java

```diff
@@ -684,6 +716,7 @@ public String serialize() {
 		containerElement.setAttribute(TargetDefinitionPersistenceHelper.ATTR_INCLUDE_ALL_PLATFORMS, Boolean.toString(getIncludeAllEnvironments()));
 		containerElement.setAttribute(TargetDefinitionPersistenceHelper.ATTR_INCLUDE_SOURCE, Boolean.toString(getIncludeSource()));
 		containerElement.setAttribute(TargetDefinitionPersistenceHelper.ATTR_INCLUDE_CONFIGURE_PHASE, Boolean.toString(getIncludeConfigurePhase()));
+		containerElement.setAttribute(TargetDefinitionPersistenceHelper.ATTR_INCLUDE_REFERENCES, Boolean.toString(getIncludeReferences()));
```

Comment:
Since this option is for a relatively special use-case (I assume most users are happy with the default true) I think the attribute should only be written when its value is not the default (i.e. if it is false).
Otherwise all existing target definitions are changed as soon as they get touched after this change has landed. And since the list of options is already lengthy, I think it should not be made even longer.